apertium-eng.eng.rlx

```diff
@@ -212,6 +217,54 @@ SELECT N IF (0 N) (0 Adj) (1 V) ;
 
 SELECT Adj IF (0 N) (0 Adj) (1 N) ;
 
+# Different cases for as ("as __ as")
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adv) (-3 Vbser) ;
+#Example (as):
+#You are as quick as I am.
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adv) (-4 Vbser) ;
+#Example (as):
+#The Earth was formed as early as 4.54 billion years ago.
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adj) (-3 Vbser) ;
+#Example (as):
+#You are as nice as I am.
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adj) (-4 Vbser) ;
+#Example (as):
+#You are not as evil as I am.
+
+# Has (Cnjsub) and has (Det)
+
+SELECT Cnjsub IF (0 Det) (-1 Adj) (-2 Vbser) (1 Nsg) ;
+#Example (that):
+#It is true that candy is evil.
+
+SELECT Cnjsub IF (0 Det) (-1 Vbser) (-2 N) (1 Nsg) ;
+#Example (that):
+#The problem is that candy is evil.
```

Comment:
What about "He thinks that candy is evil."

Reply:
I can add a rule for that.